woocommerce/woocommerce-gutenberg-products-block
# Split Checkout block component into smaller files

The component for the Checkout block kept growing and growing as we added features, it was over 440 lines of code and it was becoming difficult to work with. This PR splits it into smaller files.

Even though the line diff of this PR is quite big, it doesn't add or remove any lines other than moving files/code around, renaming some CSS classes and updating the imports according to the new file structure.

### How to test the changes in this Pull Request:

Do some smoke testing of the Checkout block: add it in the editor, change some attributes, verify it works in the frontend, try making a purchase, etc.
